app/routes/events.tsx

```diff
@@ -0,0 +1,21 @@
+import { Card } from '~/components/ui/containers';
+import { useEvents } from '~/hooks/useEvents';
+
+export default function EventsPage() {
+  const events = useEvents();
+
+  const deserializedEvents = events?.map((event) => ({
```

Comment:
Maybe we could move this into the `useEvents` hook? I could imagine we always want to work on the de-serialized version so that our backend Event type and frontend Event type match!